src/utils.py

```diff
@@ -1943,23 +1946,25 @@ def insert_htmlbox(
     # ----------------------------------------------------------------
     # Find a scaling factor that lets our story fit in
     # ----------------------------------------------------------------
-    if scale is True:
-        scale_max = None
-    else:
+    if scale_low == 1:
         scale_max = 1
+    elif scale_low > 0:
+        scale_max = 1 / scale_low
+    else:
+        scale_max = None
 
     fit = story.fit_scale(temp_rect, scale_min=1, scale_max=scale_max)
 
     if fit.big_enough is False:  # there was no fit
-        return (-1, 0)
+        return (-1, scale_low)
```

Comment:
Not 100% sure, but i thought the docs said we would return (-1, 0) in this case?

Reply:
I changed this accordingly